CHANGELOG.md

```diff
@@ -34,7 +34,7 @@ _None._
 
 ### Breaking Changes
 
-_None._
+- Decouple `PluginDirectoryServiceRemote` from Alamofire. [#725]
```

Comment:
```suggestion
- Decouple `PluginDirectoryServiceRemote` from Alamofire. [#725]
- Remove `Endpoint`. [#725]
```

`Endpoint` might have been unused, but it was still a `public` type. Do you think it would be worth to track it in the changelog?

Reply:
Sure thing. Added in https://github.com/wordpress-mobile/WordPressKit-iOS/pull/725/commits/dce379835de69e333a9748a481ab380c069fbd41.